pkg/alpine/mirrors/3.16/community

```diff
@@ -1,22 +1,25 @@
 go
+libressl-dev
+libressl3.5-libcrypto
+libressl3.5-libssl
+libressl3.5-libtls
```

Comment:
I'm a bit concerned about depending on another ssl/crypto library. Which package(s) require libresll? Shouldn't openssl suffice?

Reply:
Will check. Right now those lines in diff are mostly about moving from main to comminity repo.

Reply:
I can see that we use it here https://github.com/lf-edge/eve/blob/0c6de671a3a66ede7a3ea2d417397d08b7897a6f/pkg/xen/Dockerfile#L5
and here
https://github.com/lf-edge/eve/blob/0c6de671a3a66ede7a3ea2d417397d08b7897a6f/pkg/acrn-kernel/Dockerfile#L5
I can try to build them with openssl if you suggest.

Reply:
I see we already have it (my eyes might make missed it was a move).
So we can explore replacing it with openssl in a separate PR.